packages/flame_texturepacker/lib/src/texture_packer_atlas.dart

```diff
@@ -114,7 +141,9 @@ Future<_TextureAtlasData> _parse(
   if (fromStorage) {
     fileAsString = await File(path).readAsString();
   } else {
-    fileAsString = await Flame.assets.readFile('images/$path');
+    fileAsString = await Flame.assets.readFile(
+      '${assetsPrefix ?? 'images/'}$path',
+    );
```

Comment:
```suggestion
    assert(
      assetsPrefix != null,
      'When reading from storage, the assetsPrefix needs to be provided.',
    );
    fileAsString = await Flame.assets.readFile('$assetsPrefix!/$path');
```
To avoid having defaults on multiple places and also it's a faulty state to end up in this `else` case without an `assetsPrefix`.